pwa/app/components/tba/teamEventAppearance.tsx

```diff
@@ -53,11 +76,172 @@ export default function TeamEventAppearance({
+            <dt className="font-medium text-gray-900">Rank</dt>
+            <dd className="text-gray-700 sm:col-span-2">
+              {status.qual.ranking.rank} / {status.qual.num_teams}
+            </dd>
+          </div>
+        )}
+
+        {status?.qual?.ranking?.record && (
+          <div className="grid grid-cols-1 gap-1 py-3 sm:grid-cols-3 sm:gap-4">
+            <dt className="font-medium text-gray-900">Record</dt>
+            <dd className="text-gray-700 sm:col-span-2">
+              {status.qual.ranking.record.wins +
+                (status.playoff?.record?.wins ?? 0)}
+              -
+              {status.qual.ranking.record.losses +
+                (status.playoff?.record?.losses ?? 0)}
+              -
+              {status.qual.ranking.record.ties +
+                (status.playoff?.record?.ties ?? 0)}
+            </dd>
+          </div>
+        )}
+
+        {awards.length > 0 && (
+          <div className="grid grid-cols-1 gap-1 py-3 sm:grid-cols-3 sm:gap-4">
+            <dt className="font-medium text-gray-900">
+              {pluralize(awards.length, 'Award', 'Awards', false)}
+            </dt>
+            <dd className="text-gray-700 sm:col-span-2">
+              <ul
+                className={cn({
+                  'list-none': awards.length == 1,
+                  'list-disc': awards.length > 1,
+                })}
+              >
+                {awards.map((award) => {
+                  const namedRecipients = award.recipient_list.filter(
+                    (r) =>
+                      r.awardee !== null &&
+                      r.awardee !== '' &&
+                      r.team_key === team.key,
+                  );
+
+                  return (
+                    <li key={`${award.award_type}-${award.event_key}`}>
+                      {award.name}
+                      {namedRecipients.length > 0 ? (
+                        <>
+                          {' '}
+                          ({namedRecipients
+                            .map((r) => r.awardee)
+                            .join(', ')})
+                        </>
+                      ) : (
+                        <></>
+                      )}
+                    </li>
+                  );
+                })}
+              </ul>
+            </dd>
+          </div>
+        )}
+
+        {status?.alliance && maybeAlliances && maybeAlliances.length > 0 && (
+          <div className="grid grid-cols-1 gap-1 py-3 sm:grid-cols-3 sm:gap-4">
+            <dt className="font-medium text-gray-900">
+              {status.alliance.name}
+            </dt>
+            <dd className="text-gray-700 sm:col-span-2">
+              {joinComponents(
+                maybeAlliances
+                  .find((a) => a.picks.includes(team.key))
+                  ?.picks.map((k) => (
+                    <Link
+                      key={k}
+                      to={`/team/${k.substring(3)}`}
```

Comment:
`.substring(3)` might break for "B" teams? On prod we also strip the "B" out and just link to the base team.

Reply:
Ugh, we really just need a `<TeamLink />` component to handle things

Reply:
I don't think this needs to be dealt with now, but we need to do a pass on B team handling at some point.

Reply:
I have a solve nearly done